src/plays/personal-profile-card/components/profile-form.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react'
+
+import ProfileType from '../types'
+
+interface Props {
+  value: ProfileType
+  profile: ProfileType
+  onChange: React.ChangeEventHandler
+  onClick: () => void
+  onUpload: React.ChangeEventHandler
+  onClear: () => void
+}
+
+const ProfileForm = ({
+  value,
+  profile,
+  onChange,
+  onClick,
+  onUpload,
+  onClear,
+}: Props) => {
+  return (
+    <div className="profile-form md:mr-5">
```

Comment:
add play name as prefix of your className as mentioned for index.css file

Reply:
Fixed.